client/jetpack-connect/woo-loader-stage-images/OpeningImage.tsx

```diff
@@ -0,0 +1,91 @@
+const OpeningImage = ( props ) => (
```

Comment:
I think we can remove `props` or we should add the type hint for the component.

```suggestion
const OpeningImage = ( props: React.SVGProps< SVGSVGElement > ) => (
```